crates/astria-conductor/Cargo.toml

```diff
@@ -41,6 +41,8 @@ sequencer-client = { package = "astria-sequencer-client", path = "../astria-sequ
 ] }
 sequencer-validation = { package = "astria-sequencer-validation", path = "../astria-sequencer-validation" }
 telemetry = { package = "astria-telemetry", path = "../astria-telemetry" }
+tryhard = "0.5.1"
+humantime.workspace = true
```

Comment:
nit: use same format as used elsewhere ie
```
humantime = { workspace = true }
```

I think this may be causing the CI failure even though it shouldn't as well.